libs/designsystem/src/lib/components/checkbox/checkbox.component.ts

```diff
@@ -22,15 +22,17 @@ export class CheckboxComponent {
 
   @HostBinding('class.has-label')
   @Input()
-  labelText: string;
+  text: string;
```

Comment:
Does this work with a string? 🤔 (I guess so because of js truthy?=

Reply:
Yes. '' and undefined are falsy and does not add class